clients/es-integration/build.gradle

```diff
@@ -34,10 +34,13 @@ dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-data-elasticsearch'
     implementation 'org.springframework.cloud:spring-cloud-starter-config'
     implementation 'org.springframework.cloud:spring-cloud-starter-netflix-eureka-client'
+    implementation 'org.springframework.retry:spring-retry:1.2.4.RELEASE'
+    implementation 'org.springframework.boot:spring-boot-starter-aop:2.1.8.RELEASE'
```

Comment:
do you need to specify dependencies versions or it's already exists in spring-boot-dependencies?